Service/HandleAssentService.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace CommonGateway\HuwelijksplannerBundle\Service;
+
+use App\Entity\ObjectEntity;
+use App\Exception\GatewayException;
+use App\Service\ObjectEntityService;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\PersistentCollection;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This service holds al the logic for approving or requesting a assent.
+ */
+class HandleAssentService
+{
+    private EntityManagerInterface $entityManager;
+    private SymfonyStyle $io;
+    private ObjectEntityService $objectEntityService;
+
+    private array $data;
+    private array $configuration;
+
+    /**
+     * @param ObjectEntityService    $objectEntityService
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(
+        ObjectEntityService $objectEntityService,
+        EntityManagerInterface $entityManager
+    ) {
+        $this->objectEntityService = $objectEntityService;
```

Comment:
You never use this objectEntityService service in this file?
And you should not because this service is old and depricated